src/gui/previewselectdialog.cpp

```diff
@@ -145,6 +151,25 @@ void PreviewSelectDialog::previewButtonClicked()
     accept();
 }
 
+void PreviewSelectDialog::displayColumnHeaderMenu(const QPoint &)
```

Comment:
AFAIK, you can connect signal to a slot with less parameters count so you don't need to have such an unused QPoint.

Reply:
>AFAIK, you can connect signal to a slot with less parameters count so you don't need to have such an unused QPoint.

Right, but which would we prefer? Leave it empty? or leave the variable unnamed? The latter would match better with the signal signature.

Reply:
I would prefer to omit unused parameters.
But another question is why do you use `QCursor::pos()` instead of position provided by caller side (signal in this case)?

Reply:
>But another question is why do you use QCursor::pos() instead of position provided by caller side (signal in this case)?

Old code use it. They refer to different coordinates.

Reply:
> I would prefer to omit unused parameters.

The point is that "good slot" is just a regular method. Someone can connect it to the signal or call it directly and slot/method should behave correctly regardless. But it doesn't look like a valid method if it declares a parameter that is obviously not used (I'm not talking about overriden methods - this is another case).

Reply:
>I would prefer to omit unused parameters.

OK, added another commit to clean it up.